backend/api/Services/AreaService.cs

```diff
@@ -0,0 +1,247 @@
+﻿using Api.Controllers.Models;
+using Api.Database.Context;
+using Api.Database.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace Api.Services
+{
+    public interface IAreaService
```

Comment:
There are several functions in this class which have not been included in the interface. There are also some functions that relate to reading other entities than area, for example asset.

I suggest creating individual services for `AssetService`, `DeckService` and `InstallationService`. These files will be short but have the functionality to `ReadByName` and `ReadById` plus `Create` and `Delete`. These also need endpoints for create and delete. 

Then you can use these services in this file to reduce the complexity here. Also, the creation of these models will be handled outside the `AreaController` but then in separate controllers. More files, more lines but easier to read in every file.